generators/common/index.js

```diff
@@ -72,18 +72,42 @@ module.exports = class extends BaseBlueprintGenerator {
     }
 
     // Public API method used by the getter and also by Blueprints
-    _default() {
+    _loading() {
         return {
             loadSharedConfig() {
                 this.loadAppConfig();
                 this.loadClientConfig();
                 this.loadServerConfig();
                 this.loadTranslationConfig();
             },
-            setupSharedOptions() {
+        };
+    }
+
+    get loading() {
```

Comment:
I think `loadingConfig` would be more explicit

Reply:
I am not sure that would make much difference.
Probably we should add a good description at `generator-blueprint.js` and the blueprint generator.
But if I am open for the change.